packages/main/src/RPA/Robocorp/WorkItems.py

```diff
@@ -1240,25 +1291,33 @@ def get_work_item_payload(self):
 
         .. code-block:: robotframework
 
-            ${payload}=    Get work item payload
-            Log    Entire payload as dictionary: ${payload}
+            *** Tasks ***
+            Example task
+                ${payload}=    Get work item payload
+                Log    Entire payload as dictionary: ${payload}
         """
         return self.current.payload
 
     @keyword
     def set_work_item_payload(self, payload):
-        """Set the full JSON payload for a work item.
+        r"""Set the full JSON payload for a work item.
 
         :param payload: Content of payload, must be JSON-serializable
 
         **NOTE**: Most use cases should prefer higher-level keywords.
+        Using this keyword may cause errors when getting the payload via
```

Comment:
Very good point here! (as I've seen the behaviour, but not sure exactly what other kind of payload you may have, like a list of dicts or?)

Reply:
Or just a list, yes. Payload needs to have a `dict` as it's only contents or the `Get work item variable` keyword will fail because that just pulls a key off the `dict` attached to `Payload`.